src/pages/[platform]/build-a-backend/graphqlapi/set-up-graphql-api/index.mdx

```diff
@@ -465,6 +465,6 @@ There's so much more to discover with Amplify's GraphQL API capabilities. Learn
 - [Add custom business logic to the GraphQL API](/[platform]/build-a-backend/graphqlapi/custom-business-logic/)
 - [Run search and result aggregation queries](/[platform]/build-a-backend/graphqlapi/search-and-result-aggregations/)
 - [Connect to machine learning services](/[platform]/build-a-backend/graphqlapi/connect-machine-learning-services/)
-- [Examples and solutions](/[platform]/build-a-backend/graphqlapi/best-practice/)
+- [Examples and solutions](/javascript/build-a-backend/graphqlapi/best-practice/)
```

Comment:
Isn't this page available on other platforms?

Reply:
Yes all JS family pages can use this page and swift - I would suggest we either take it out from this page entirely or use the inline filter vs make a direct link. @renebrandel could we remove this link from the Set up page? If anything I think we would want to check the links on reading data and make sure this has a link from there right or set up with a filter